rust-bot/src/commands/mod_group.rs

```diff
@@ -22,6 +22,10 @@ use serenity::{
 #[commands(create_cohort)]
 struct Mod;
 
+// Party Corgi - Mod Role Id = 639531892437286959
+const MOD_ROLE_ID: u64 = 639531892437286959;
```

Comment:
I could have this be:
```rs
const MOD_ROLE_ID: RoleId = 639531892437286959.into();
```
So in many call sites you'll see me change do this with `let` instead.